proto_compat_test.go

```diff
@@ -21,7 +23,7 @@ func TestReadCompatProtoData(t *testing.T) {
 
 	cases := []struct {
 		protoType string
-		want      proto.Message
+		want      protoreflect.ProtoMessage
```

Comment:
Why protoreflect.ProtoMessage and not proto.Message?

Reply:
Changed